playbook/roles/varnish/templates/default.vcl.j2

```diff
@@ -129,7 +129,12 @@ sub vcl_recv {
   }
   {% for director in varnish.directors %}
   elsif (req.http.host ~ "{{ director.host }}"){
+    {% if varnish.client_ip_sticky_session %}
+    #Use client ip has seed for backend.
+    set req.backend_hint = {{ director.name }}.backend(client.ip);
```

Comment:
This would be better to handle around lines 114:116 where the sticky property is set.